data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/RdsSourceConfig.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.source.rds;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+import org.opensearch.dataprepper.plugins.source.rds.configuration.AwsAuthenticationConfig;
+import org.opensearch.dataprepper.plugins.source.rds.configuration.EngineType;
+import org.opensearch.dataprepper.plugins.source.rds.configuration.ExportConfig;
+
+import java.util.List;
+
+/**
+ * Configuration for RDS Source
+ */
+public class RdsSourceConfig {
+
+    /**
+     * Identifier for RDS instance or Aurora cluster
+     */
+    @JsonProperty("db_identifier")
+    private String dbIdentifier;
+
+    @JsonProperty("engine")
+    private EngineType engine = EngineType.MYSQL;
+
+    @JsonProperty("tables")
+    private List<String> tables;
```

Comment:
Do you think we will need additional configurations for each table? Maybe these should be objects. An alternative is to rename this to `table_names`. Then we have flexibility to add objects in the future.

Reply:
I'm not sure at this time. Changing it to `table_names` makes sense. We can review the naming again before release.

Reply:
Renamed.